lora-phy/Cargo.toml

```diff
@@ -20,9 +20,9 @@ lorawan-device = { path = "../lorawan-device", version = ">=0.11.0", features =
     "async",
 ], optional = true }
 num-traits = { version = "0.2", default-features = false }
-
-embedded-hal = { version = "=1.0.0-rc.2" }
-embedded-hal-async = { version = "=1.0.0-rc.2" }
+embedded-hal = { version = "=1.0.0" }
```

Comment:
no need to use `=` anymore, neer `1.x.x` releases are now guaranteed to be backwards-compatible.